### PR TITLE
DR-748 Remove unneeded code and scripts

### PR DESCRIPTION
**Pull request by NamrataSweya**
removed unneeded script

### CODE REVIEW

File: common/config/rush/command-line.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
       "commandKind": "global",
       "name": "start:dev",
       "summary": "Runs API in dev mode",
-      "shellCommand": "concurrently -n ganache,backend \"(cd apps/drec-api && rushx run:ganache)\" \"(cd apps/drec-api && rushx start:dev)\" \"(cd apps/drec-ui && rushx start:dev)\"",
+      "shellCommand": "concurrently -n ,backend \"(cd apps/drec-api && rushx start:dev)\"",
```
Comment (dmohns) [inline common/config/rush/command-line.json]: I don't think you need `concurrently` here anymore. You are just running one script 😉

**Comment by dmohns** [comment]
I guess you should also remove the `concurrently` dependency now, right?

**Comment by dmohns** [comment]
Please don't remove `integrators-scripts` as part of this Pull Request. I have conflicting statements about whether or not `integrators-scripts` are still helpful for setting up a development environment.

**Comment by NamrataSweya** [comment]
> Please don't remove `integrators-scripts` as part of this Pull Request. I have conflicting statements about whether or not `integrators-scripts` are still helpful for setting up a development environment.

ok ,
let me commit again

**Comment by dmohns** [comment]
Hey @NamrataSweya 
I saw you added back a lot of the `integrator-scripts`. However 

1. Not all of them are added back.
2. Some of the files are empty.

Could you please make it so, that this PR doesn't show any diff in the `integrator-scripts` directory.

**Pull request closed without merging**